backtester/report/report.go

```diff
@@ -0,0 +1,171 @@
+package report
+
+import (
+	"fmt"
+	"html/template"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/backtester/common"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/kline"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+// GenerateReport sends final data from statistics to a template
+// to create a lovely final report for someone to view
+func (d *Data) GenerateReport() error {
```

Comment:
Would like to add an output for initial funds (1,000,000) - final funds (918383.32) for example which would give you the strategies committed funds (81,616.68) then we can calculate the strategies return based on committed funds and not the overall portfolio or supplied funds.